core/src/test/scala/org/apache/spark/io/ChunkedByteBufferSuite.scala

```diff
@@ -56,6 +56,12 @@ class ChunkedByteBufferSuite extends SparkFunSuite {
     assert(chunkedByteBuffer.getChunks().head.position() === 0)
   }
 
+  test("writeFully() does not affect original buffer's position") {
```

Comment:
Hi @manbuyun .You should add a new unit test to support your own change. For example, "writeFully() can write buffer which is larger than `bufferWriteChunkSize` correctly. " And update the test code.

Reply:
Done. Thanks